packages/solid-query/src/__tests__/createQuery.test.tsx

```diff
@@ -1579,7 +1579,6 @@ describe('createQuery', () => {
 
     // Fetch query
     expect(states[0]).toMatchObject({
-      data: undefined,
```

Comment:
Is it important that this property be defined on the original state object (e.g. when we `states.push({ ...state })`)? When accessing through the proxy, e.g. `state.data`, it will come back as undefined as expected.

Reply:
This fine for now! I think libraries built on top of solid-query would need to know about this caveat. That's worth a consideration